api/ruleset.go

```diff
@@ -71,7 +75,16 @@ func (h RuleSetHandler) List(ctx *gin.Context) {
 		h.DB(ctx),
 		clause.Associations,
 		"Rules.File")
-	result := db.Find(&list)
+
+	filter, err := qf.New(ctx,
+		[]qf.Assert{
+			{Field: "labels", Kind: qf.STRING},
```

Comment:
Adding `name` to the filter in the interest of completeness _may_ be a nice to have but can always add later.